d2l-floating-buttons.html

```diff
@@ -40,17 +40,17 @@
 				padding: 0.75rem 0 0 0;
 				position: relative;
 			}
```

Comment:
I guess there can't be anything in between the selector and the `::slotted`?

Reply:
Yeah, I was running into troubles.  I saw some references to the shim translating `x ::slotted(y)` into something like `x > y`.